regression-test/suites/load/broker_load/test_array_load.groovy

```diff
@@ -116,13 +137,30 @@ suite("test_array_load", "load") {
         assertTrue(result1[0][0] == 0, "Query OK, 0 rows affected")
     }
     
+    def check_load_result = {checklabel, testTablex ->
+        max_try_micro_secs = 10000
+        while(max_try_micro_secs--) {
+            result = sql "show load where label = '${checklabel}'"
+            if(result[0][2] == "FINISHED") {
+                qt_select "select * from ${testTablex} order by k1"
+                break
+            } else {
+                sleep(1000) // wait 1 second every time
+                max_try_micro_secs = max_try_micro_secs - 1000
+                if(max_try_micro_secs < 0) {
```

Comment:
why not max_try_micro_secs <= 0

Reply:
if no need "max_try_micro_secs--", I think it should be 'max_try_micro_secs <= 0'.